compass/ocean/tests/hurricane/mesh/dequ120at30cr10rr2/__init__.py

```diff
@@ -11,15 +11,12 @@ def build_cell_width_lat_lon(self):
         """
         Create cell width array for this mesh on a regular latitude-longitude
         grid
-
         Returns
         -------
         cellWidth : numpy.array
             m x n array of cell width in km
-
         lon : numpy.array
             longitude in degrees (length n and between -180 and 180)
-
         lat : numpy.array
             longitude in degrees (length m and between -90 and 90)
```

Comment:
I would leave the blank lines here as they were.  That's our formatting convention for docstrings.

Reply:
Done.